cmd/legacy/run.go

```diff
@@ -100,7 +102,26 @@ func Run(v *viper.Viper) {
 		todaygoal.Run(v)
 	}
 
-	log.Debugln("command: heartbeat")
+	if v.IsSet("entity") {
+		log.Debugln("command: heartbeat")
 
-	heartbeatcmd.Run(v)
+		heartbeatcmd.Run(v)
+	}
+
+	if v.IsSet("sync-offline-activity") {
+		log.Debugln("command: sync-offline-activity")
+
+		offlinesync.Run(v)
```

Comment:
Does `Run` stop execution? If not, we would always log fatal below.

Reply:
yep it has its own exit.